DEVELOPER_SETUP.md

```diff
@@ -0,0 +1,110 @@
+# Developer setup guide
+
+## Requirements
+
+To run this project, you'll need `python`, `pip` and `poetry` installed.
+
+```
+python = <=3.11, <3.12
+```
+
+## Installing packages
+
+Currently we use [poetry](https://python-poetry.org/) to manage our python packages. The list of poetry groups and python packages we install can be found [here](https://github.com/i-dot-ai/redbox-copilot/blob/main/pyproject.toml) in `pyproject.toml`.
+
+To install the packages, you'll first need to create a new virtual environment for python, and then run the following in that virtual environment.
+
+``` bash
+poetry install
+```
+
+## Setting environment variables
+
+We use `.env` files to populate the environment variables for local development. When cloning the repository the files `.env.test`, `.env.django`, `.env.integration` and `.env.example` will be populated.
+
+To run the project, create a new file called `.env` and populate this file with the setting names from `.env.example` and the values these settings need.
+
+**`.env` is in `.gitignore` and should not be committed to git**
+
+## Building and running the project
+
+To view all of the build commands, check the `Makefile` that can be found [here](https://github.com/i-dot-ai/redbox-copilot/blob/main/Makefile).
+
+The project currently consists of multiple docker images needed to run the project in it's entirety. If you only need a subsection of the project running, for example if you're only editing the django app, you can run a subset of the images. The images currently in the project are:
+
+- `elasticsearch`
+- `kibana`
+- `embedder`
+- `ingester`
+- `minio`
+- `miniocreatebuckets`
```

Comment:
```suggestion
- `miniocreatebuckets`
```
we dont actually need this anymore

Reply:
Ah fair enough, I just took it from the list in `make run`, am I ok to remove it from that command too?